bin/deploy

```diff
@@ -36,5 +36,14 @@ aws s3 sync \
 
 echo "Deployed styleguide to http://rcl.policygenius.com/$VERSION/index.html"
 
+echo '+++ Deploying styleguide latest to S3'
+aws s3 sync \
+  ./styleguide s3://rcl-pg/latest/ \
```

Comment:
problem 1: you're telling browsers to cache the result of this page for a very long time, which means they won't be getting the latest version latest
problem 2: this S3 bucket has a cloudfront distro in front of it and we'd need to manually invalidate their cache (which costs money and is slow) every time we deploy a new thingy

if you resolve those 2 problems, it'd probably work